### PR TITLE
Changed tabular data wrangling notebooks

### PR DESCRIPTION
**Pull request by marabuuu**
Hi again @haesleinhuepf,

Here another one for modified __tabular data wrangling__ notebooks:
- Handling NaNs
- Introduction to DataFrames
- Select rows and columns
where I tried to simplify things or make them more visually appealing. I also added a link to a pandas cheat sheet

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -20,3 +20,4 @@ schematics/Jaccard_index_binary.png
 docs/29_algorithm_validation/ideas.ipynb
 docs/29_algorithm_validation/solution for exercise - metrics to investigate segmentation results.ipynb
 *.png
+*.csv
```
Comment (haesleinhuepf) [inline .gitignore:23]: Ah, we should not ignore all CSV files. Can you change that to the specific csv files you would like to have ignored?

**Review by haesleinhuepf — CHANGES_REQUESTED** [review]
Fantastic, thanks @marabuuu ! 

Two little things:
* Can you add your notebooks in the right chapter in `docs/_toc.yml`?
* Let's not ignore csv files in general (see comment below).

Big thanks again!

**Comment by marabuuu** [comment]
Thanks @haesleinhuepf for the hint, it's done now. I have changed the _toc.yml in PR #31 as it is related to feature extraction. 

Best,
Mara

**Comment by haesleinhuepf** [comment]
Awesome, thanks @marabuuu, also for fixing all my typos 🙊

**Pull request merged**